content/accessibility/tools.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: Accessibility tools
+---
+
+A selection of tools to help product designers design accessibly. 
+
+## Figma 
+
+### [Accessibility sticker sheet](https://www.figma.com/file/o3Vhxw4SmBsUjeaVq3nF4y/Accessibility-Notation?node-id=0%3A1)
+This Figma file includes annotations to use on your designs prior to developer hand off. These annotations let you note accessibility information like desired tab order, screen reader labels, headings, and landmarks. Pull directly from the file or turn on the accessibility annotations in your Figma assets list.
+
+![""](https://user-images.githubusercontent.com/40274682/89364293-04f02f00-d687-11ea-9542-439fa61ef3c8.png)
+
+### [A11y - Focus Orderer plugin](https://www.figma.com/community/plugin/731310036968334777/A11y---Focus-Orderer)
+
+Select your elements in the desired tabbing order, add them to the plugin to get a visible layer label, and use the tab key to test the tab order from inside of Figma. 
+- **How does this help?** Instead of manually placing tab order stickers on your design, let the plugin do it for you, and you can tab through your design to see if the flow makes sense.
```

Comment:
```suggestion
**How does this help?** Instead of manually placing tab order stickers on your design, let the plugin do it for you, and you can tab through your design to see if the flow makes sense.
```